### ISSUE TITLE
When running to Triplane Gaussian Transformers(TGS), crashes and crashes occur without logs.

### ISSUE THREAD

**Issue by WorkSync** [MrForExample/ComfyUI-3D-Pack#236]
Hello everyone, I have tested the workflow——Triplane_Gaussian_Transformers_to_3DGS_and_3DMesh.json.What I found when I was testing (I had the model ready) was when I ran to the node——crashes and crashes occur without logs.
At this point, the console displays the following information：
> To see the GUI go to: http://127.0.0.1:8188
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
> [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
> FETCH DATA from: C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
> FETCH DATA from: C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
> got prompt
> TriplaneGaussian.models.tokenizers.image
> TriplaneGaussian.models.pointclouds.simplepoint
> TriplaneGaussian.models.tokenizers.image
> TriplaneGaussian.models.networks
> TriplaneGaussian.models.tokenizers.point
> TriplaneGaussian.models.transformers
> TriplaneGaussian.models.networks
> TriplaneGaussian.models.snowflake.model_spdpp
> TriplaneGaussian.models.pointclouds.pointnet
> [Comfy3D] [Load_Triplane_Gaussian_Transformers] loaded model ckpt from C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190\custom_nodes\ComfyUI-3D-Pack\Checkpoints\TriplaneGaussian\model_lvis_rel.ckpt
> Warn!: C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190\custom_nodes\ComfyUI-3D-Pack\Gen_3D_Modules\TriplaneGaussian\data.py:91: UserWarning: Using torch.cross without specifying the dim arg is deprecated.
> Please either pass the dim explicitly or simply use torch.linalg.cross.
> The default value of dim will change to agree with that of linalg.cross in a future release. (Triggered internally at ..\aten\src\ATen\native\Cross.cpp:66.)    
>   right: Float[Tensor, "B 3"] = F.normalize(torch.cross(lookat, up), dim=-1)    
> 
> Warn!: C:\ZTData\Database\anaconda_env\comfyui_2\Lib\site-packages\diffusers\models\attention_processor.py:1584: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
>   hidden_states = F.scaled_dot_product_attention(
> 
> PS C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190>

I‘ve tested [the original project of Triplane_Gaussian_Transformers](https://github.com/VAST-AI-Research/TriplaneGaussian),I found that when I was running this native project, there were crashes and crashes, and during the run, I noticed that the CPU was working but the GPU was not.As shown in the following figure：
![QQ_1721796819804](https://github.com/user-attachments/assets/4d1d595e-b65b-4679-91b2-aacdb8a1d18e)

But I am certain that CUDA is installed and there are no conflicts with its dependencies.(Actually, I tested the workflow——Tripo3D and successfully generated the 3D model.When it comes to Triplane_Gaussian_Transformers,it failed,which makes me sad.I've been stuck on this problem for a week.):

> (C:\ZTData\Database\anaconda_env\comfyui_2) C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190>nvcc --version
> nvcc: NVIDIA (R) Cuda compiler driver
> Copyright (c) 2005-2023 NVIDIA Corporation
> Built on Mon_Apr__3_17:36:15_Pacific_Daylight_Time_2023
> Cuda compilation tools, release 12.1, V12.1.105
> Build cuda_12.1.r12.1/compiler.32688072_0

And the pip list:

> (C:\ZTData\Database\anaconda_env\comfyui_2) C:\ZTData\Database\Code\Code_Py\clone_projects\ComfyUI_8190>pip list
> DEPRECATION: Loading egg at c:\ztdata\database\anaconda_env\comfyui_2\lib\site-packages\pointnet2_ops-3.0.0-py3.11-win-amd64.egg is deprecated. pip 24.3 will enforce this behaviour change. A possible replacement is to use pip for package installation.. Discussion can be found at https://github.com/pypa/pip/issues/12330Package                     Version
> --------------------------- ------------------
> aiofiles                    23.2.1
> aiohttp                     3.9.5
> aiosignal                   1.3.1
> annotated-types             0.7.0
> antlr4-python3-runtime      4.9.3
> anyio                       4.4.0
> attrs                       23.2.0
> certifi                     2024.7.4
> cffi                        1.16.0
> charset-normalizer          3.3.2
> click                       8.1.7
> cmake                       3.30.1
> colorama                    0.4.6
> coloredlogs                 15.0.1
> contourpy                   1.2.1
> cryptography                43.0.0
> cycler                      0.12.1
> dataclasses-json            0.6.7
> Deprecated                  1.2.14
> diff_gaussian_rasterization 0.0.0
> diffusers                   0.29.2
> dnspython                   2.6.1
> einops                      0.8.0
> email_validator             2.2.0
> executing                   2.0.1
> fastapi                     0.111.1
> fastapi-cli                 0.0.4
> ffmpy                       0.3.2
> filelock                    3.15.4
> flash_attn                  2.6.1
> flatbuffers                 24.3.25
> fonttools                   4.53.1
> frozenlist                  1.4.1
> fsspec                      2024.6.1
> ftfy                        6.2.0
> fvcore                      0.1.5.post20221221
> gitdb                       4.0.11
> GitPython                   3.1.43
> gradio                      4.39.0
> gradio_client               1.1.1
> gradio_model3dgs            0.2.5
> h11                         0.14.0
> httpcore                    1.0.5
> httptools                   0.6.1
> httpx                       0.27.0
> huggingface-hub             0.24.0
> humanfriendly               10.0
> idna                        3.7
> imageio                     2.34.2
> imageio-ffmpeg              0.5.1
> importlib_metadata          8.0.0
> importlib_resources         6.4.0
> intel-openmp                2021.4.0
> iopath                      0.1.10
> jaxtyping                   0.2.33
> Jinja2                      3.1.4
> jsonschema                  4.23.0
> jsonschema-specifications   2023.12.1
> kiui                        0.2.10
> kiwisolver                  1.4.5
> kornia                      0.7.3
> kornia_rs                   0.1.5
> lazy_loader                 0.4
> lightning-utilities         0.11.5
> llvmlite                    0.43.0
> markdown-it-py              3.0.0
> MarkupSafe                  2.1.5
> marshmallow                 3.21.3
> matplotlib                  3.9.1
> matrix-client               0.4.0
> mdurl                       0.1.2
> mkl                         2021.4.0
> mpmath                      1.3.0
> msvc_runtime                14.40.33807
> multidict                   6.0.5
> mypy-extensions             1.0.0
> nerfacc                     0.5.3
> networkx                    3.3
> ninja                       1.11.1.1
> numba                       0.60.0
> numpy                       1.26.4
> nvdiffrast                  0.3.1
> objprint                    0.2.3
> omegaconf                   2.3.0
> onnxruntime                 1.18.1
> open_clip_torch             2.26.1
> opencv-python               4.10.0.84
> opencv-python-headless      4.10.0.84
> orjson                      3.10.6
> packaging                   24.1
> pandas                      2.2.2
> piexif                      1.1.3
> pillow                      10.4.0
> pip                         24.0
> platformdirs                4.2.2
> plyfile                     1.0.3
> pointnet2_ops               3.0.0
> pointnet2_ops               3.0.0
> pooch                       1.8.2
> portalocker                 2.10.1
> protobuf                    5.27.2
> psutil                      6.0.0
> py-cpuinfo                  9.0.0
> pycparser                   2.22
> pydantic                    2.8.2
> pydantic_core               2.20.1
> pydub                       0.25.1
> PyGithub                    2.3.0
> pygltflib                   1.16.2
> Pygments                    2.18.0
> pyhocon                     0.3.61
> PyJWT                       2.8.0
> PyMatting                   1.1.12
> PyMCubes                    0.1.6
> pymeshlab                   2023.12.post1
> PyNaCl                      1.5.0
> pyparsing                   3.1.2
> pyreadline3                 3.4.1
> python-dateutil             2.9.0.post0
> python-dotenv               1.0.1
> python-multipart            0.0.9
> pytorch-lightning           2.3.3
> pytorch-msssim              1.0.0
> pytorch3d                   0.7.7
> pytz                        2024.1
> pywin32                     306
> PyYAML                      6.0.1
> referencing                 0.35.1
> regex                       2024.5.15
> rembg                       2.0.57
> requests                    2.32.3
> rich                        13.7.1
> rpds-py                     0.19.0
> ruff                        0.5.4
> safetensors                 0.4.3
> scikit-image                0.24.0
> scipy                       1.14.0
> seaborn                     0.13.2
> segment-anything            1.0
> semantic-version            2.10.0
> sentencepiece               0.2.0
> setuptools                  69.5.1
> shellingham                 1.5.4
> simple_knn                  0.0.0
> six                         1.16.0
> smmap                       5.0.1
> sniffio                     1.3.1
> soundfile                   0.12.1
> spandrel                    0.3.4
> starlette                   0.37.2
> sympy                       1.13.1
> tabulate                    0.9.0
> tbb                         2021.13.0
> termcolor                   2.4.0
> tifffile                    2024.7.21
> timm                        1.0.7
> tokenizers                  0.19.1
> tomlkit                     0.12.0
> torch                       2.3.0+cu121
> torch_scatter               2.1.2
> torchaudio                  2.3.0+cu121
> torchmetrics                1.4.0.post0
> torchsde                    0.2.6
> torchtyping                 0.1.4
> torchvision                 0.18.0+cu121
> tqdm                        4.66.4
> trampoline                  0.1.2
> transformers                4.42.4
> trimesh                     4.4.3
> triton                      2.1.0
> typeguard                   2.13.3
> typer                       0.12.3
> typing_extensions           4.12.2
> typing-inspect              0.9.0
> tzdata                      2024.1
> ultralytics                 8.2.64
> ultralytics-thop            2.0.0
> urllib3                     2.2.2
> uvicorn                     0.30.3
> varname                     0.13.3
> watchfiles                  0.22.0
> wcwidth                     0.2.13
> websockets                  11.0.3
> wheel                       0.43.0
> wrapt                       1.16.0
> xatlas                      0.0.9
> xformers                    0.0.26.post1
> yacs                        0.1.8
> yarl                        1.9.4
> zipp                        3.19.2

I would appreciate it if someone could help me qaq.

**Comment by WorkSync** [comment]
My GPU models are as follows:

> NVIDIA GeForce RTX 3060

My output code is:
```
import torch
print(torch.cuda.get_device_properties(0).name)
```

**Comment by atilladeniz** [comment]
same here

**Comment by Duodecimus** [comment]
Your torchscatter isn't installed with cuda.
Mine looks like this in pip: `torch_scatter==2.1.2+pt23cu121`

**Comment by WorkSync** [comment]
> Your torchscatter isn't installed with cuda. Mine looks like this in pip: `torch_scatter==2.1.2+pt23cu121`

Thank you for your reply, but my pip show that I have installed it and it does not match version `2.1.2+pt23cu121`
